rest-api/tools/setup_database.sh

```diff
@@ -50,33 +50,16 @@ fi
 
 source tools/setup_vars.sh
 
-# Prompts the user for a database user password.
-function get_required_password {
-  db_user_name=$1
-  echo -n "Store and share this password in Valentine with"
-  echo " name=\"$db_user_name\" and description=\"$PROJECT database\"."
-  read -s -p "$db_user_name password for database: " new_password
-  echo
-  if [ -z "${new_password}" ]
-  then
-    echo "Password required; exiting."
-    exit 1
-  fi
-
-  read -s -p "Repeat $db_user_name password: " repeat_new_password
-  echo
-  if [ "${repeat_new_password}" != "${new_password}" ]
-  then
-    echo "Password mismatch; exiting."
-    exit 1
-  fi
-}
+# Get a randomly generated password
+function randpw {
+    new_password=$(< /dev/urandom LC_CTYPE=C tr -dc _A-Z-a-z-0-9 | head -c${1:-16};echo;)
```

Comment:
Looks like you've got hyphen in there twice?

Reply:
tested, it could create 2 hyphens. removed.